src/server/templates/race.html

```diff
@@ -430,10 +441,15 @@
 
 		socket.on('first_pass_registered', function (msg) {
 			if (!rotorhazard.beep_crossing_exited && rotorhazard.beep_on_first_pass_button) {
-				play_beep(35, node_tone[msg.node_index], rotorhazard.indicator_beep_volume, 'sawtooth', 0.02);
-				setTimeout(function(tone){
-					play_beep(35, node_tone[tone], rotorhazard.indicator_beep_volume, 'sawtooth', );
-				}, 70, msg.node_index);
+				if( rotorhazard.use_mp3_tones){
+					sound_beep.play();
+				}
+				else {
+					play_beep(35, node_tone[msg.node_index], rotorhazard.indicator_beep_volume, 'sawtooth', 0.02);
+					setTimeout(function(tone){
+						play_beep(35, node_tone[tone], rotorhazard.indicator_beep_volume, 'sawtooth', );
```

Comment:
Is the trailing comma intentional?

Reply:
No Idea, thats from the original code done by HazardCreative

Reply:
Can be removed.
```suggestion
						play_beep(35, node_tone[tone], rotorhazard.indicator_beep_volume, 'sawtooth');
```